kernel/packages/shared/loading/ReportFatalError.ts

```diff
@@ -22,6 +22,7 @@ declare const globalThis: StoreContainer
 
 export let aborted = false
```

Comment:
We shouldn't export this variable if isn't used elsewhere.

Reply:
It was already being used in these 2 places:
- `BrowserInterfaces.ts` -> `ControlEvent(...)` -> when it receives the `ActivateRenderingACK` event, the `aborted` flag is checked to notify observers or not.
- `dcl.ts` -> `startUnitySceneWorkers()` -> when the event `onPositionSettled` is triggered, the `aborted` flag is checked to teleport and activate rendering or not.

Reply:
Oops! Didn't found those uses for some reason. Thanks for the explanation!